script/changelog/parser.ts

```diff
@@ -40,7 +40,7 @@ function getChangelogEntry(commit: IParsedCommit, pr: IAPIPR): string {
   let type = PlaceholderChangeType
   const description = capitalized(pr.title)
 
-  const re = /Fixes #(\d+)/gi
+  const re = /Fixes:? #(\d+)/gi
```

Comment:
Are any other characters allowed here?

Reply:
@j-f1 I think yes! See: https://help.github.com/articles/closing-issues-using-keywords/

If by characters you mean `:`, `?`, I'm not sure but for words we can definitely use more words than the word `Fixes`. More here https://help.github.com/articles/closing-issues-using-keywords/

Reply:
I knew this was going to be a can of worms...

![](https://user-images.githubusercontent.com/359239/37632556-da26d3a0-2c42-11e8-8d73-28bceb3bf16b.gif)

Reply:
Should it also support more then a single space in between colon and #

Reply:
@staabm probably, yes